docs/getting-started/user-guides/paymetheus.md

```diff
@@ -0,0 +1,60 @@
+# **Paymetheus** #
+
+Paymetheus is a simple to use graphical wallet for Decred. With it, you can send and receive DCR, 
+purchase tickets for [PoS voting](/mining/proof-of-stake.md), get a history of all your transactions and more.
+About the only thing Paymetheus doesn't do is PoS voting.
+
+---
+
+## **Download and Install** ##
+The first thing you'll need to do is get a copy of Paymetheus. You will find the downloads [here](https://github.com/decred/decred-binaries/releases). 
+Download the file for your operating system and install. Once you're done, run the 'Decred' program.
+You'll notice a program called "Paymetheus Standalone". This does not include the 'dcrd' program 
+required to connect to the Decred network, so don't use this one.
+
+---
+
+## **Start Paymetheus** ##
+You're now ready to start using Decred! Start Paymetheus Standalone. You will now see the connection screen:  
+
+![Paymetheus connection screen](../../img/Paymetheus-dcrd-login.png)  
+
+Paymetheus comes with a dcrd server built in so just press Continue.
+
+---
+
+## **Create or Restore Wallet** ##
+You will now have the option to create a new wallet or restore a wallet from a seed. If you're reading this, you're probably new and don't 
+already have a wallet so let's create a new one. If you click restore, you will be asked to enter your seed words instead of being given a new list.
```

Comment:
a new list of what exactly?

I'd just say "a new seed" instead, since that was just referenced earlier in the sentence.